### PR TITLE
Updating quick start enterprise

### PR DESCRIPTION
**Pull request by stevenGravy**
- take out comment on internal.logins.  internal.logins are used as part of trusted clusters.
- Add in logins since users have been skipping the access role part
- Use the joe user directly and mention it must exist (easy miss)
- HTTPS mention correction

### CODE REVIEW

File: docs/pages/enterprise/getting-started.mdx [hunk 1]
```diff
@@ -259,7 +250,7 @@ $ sudo tctl create -f role.yaml
 Now, lets create a new Teleport user "joe" with "access" role:
 
 ```bsh
-$ sudo tctl users add --roles=access joe
+$ sudo tctl users add --roles=access joe --logins=joe,ubuntu,ec2-user
```
Comment (zmb3) [inline docs/pages/enterprise/getting-started.mdx:253]: Can you put `--logins` before `joe`. It's easy for the argument to "get lost" when it's placed in between flags like this.

File: docs/pages/enterprise/getting-started.mdx [hunk 2]
```diff
@@ -322,8 +313,8 @@ store it in `~/.tsh/keys/<proxy>` directory.
 With a certificate in place, Joe can now interact with the Teleport cluster:
 
 ```bsh
-# SSH into any host behind the proxy:
-$ tsh ssh node.example.com
+# SSH into any host behind the proxy (joe user should already exist on the node):
```
Comment (zmb3) [inline docs/pages/enterprise/getting-started.mdx:316]:
```suggestion
# SSH into any host behind the proxy (Unix user 'joe' should already exist on the node):
```

**Review by zmb3 — APPROVED** [review]
A couple suggestions but looks good to me.

File: docs/pages/enterprise/getting-started.mdx [hunk 3]
```diff
@@ -443,7 +434,7 @@ This example command will create a Teleport user called `testuser` which has the
 `access` role. Feel free to change these to suit your needs.
 
 ```code
-$ docker exec teleport tctl users add testuser --roles=access
+$ docker exec teleport tctl users add testuser --roles=access --logins=root,ec2-user=ubuntu
```
Comment (ptgott) [inline docs/pages/enterprise/getting-started.mdx:437]: Just double-checking that final `=` should be there—I'm not sure I've seen that syntax in the value of `logins` before.

Comment (stevenGravy) [reply docs/pages/enterprise/getting-started.mdx]: yes, thanks.  corrected and made in the format of the earlier tctl users add

**Review by ptgott — APPROVED** [review]

**Pull request merged**